components/script/dom/xmlhttprequest.rs

```diff
@@ -368,6 +373,11 @@ impl XMLHttpRequestMethods for XMLHttpRequest {
                 // Step 11 - abort existing requests
                 self.terminate_ongoing_fetch();
 
+                // See WPT test: FileAPI/blob/Blob-XHR-revoke.html
```

Comment:
Feel free to make a PR to wpt to remove this test, too, since it's unspecced and inconsistent across browsers. If this gets specced and/or becomes consistent, we can add it again.

We should probably create an issue on a spec somewhere, but right now I'm discussing with baku and we'll figure out which spec we care about.